C/common/string_utils.cpp

```diff
@@ -34,3 +34,25 @@ void StringReplace(std::string& StringToManage,
 				       StringReplacement);
 	}
 }
+
+/**
+ * Strips Line feed and carige return
+ *
+ */
+void StringStripCRLF(std::string& StringToManage)
```

Comment:
I guess only a single CR & LF characters are expected in input string?

Reply:
> I guess only a single CR & LF characters are expected in input string?

we had to manage the HTTP headers the obtain a description from a HTTP request.
the headers contain CR & LF and so we used this function to strip them.